File: RepoWebShop/src/app/cart/classes/delivery-address.ts

export class DeliveryAddress {
  'addressLine1': string;
  'streetNumber': string;
  'streetName': string;
  'state': string;
  'country': string;
  'distance': number;
  'deliveryInstructions': string;
  'deliveryCost': number;
  'deliveryAddressId': number;

  'latitude': number;
  'longitude': number;

  constructor(streetNumber: string, streetName: string, lat: number, lng: number) {
    this.streetName = streetName;
    this.streetNumber = streetNumber;
    this.latitude = lat;
    this.longitude = lng;
  }

}

File: RepoWebShop/FeModels/_DeliveryAddress.cs
using RepoWebShop.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RepoWebShop.FeModels
{
    public class _DeliveryAddress
	{
		public string AddressLine1 { get; set; }
		public string StreetNumber { get; set; }
		public string StreetName { get; set; }
		public string ZipCode { get; set; }
		public string State { get; set; }
		public string Country { get; set; }
		public int Distance { get; set; }
		public string DeliveryInstructions { get; set; }
		public decimal DeliveryCost { get; set; }
		public int DeliveryAddressId { get; set; }
		public string Latitude { get; set; }
		public string Longitude { get; set; }
	}
}
